Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/static/js/tooltip.js

```diff
@@ -79,11 +80,11 @@ class _ToolTip {
                         const rightPoint = stateDiff.points.length > 1 && stateDiff.points[1].x > stateDiff.points[0].x ? stateDiff.points[1] : stateDiff.points[0];
 
                         direction = 'left';
-                        let tipX = leftPoint.x - 12 - bounds.width;
```

Comment:
This change seems unrelated to the rest of the PR.

Reply:
This is also for the new tooltip arrow